src/utils/pyworker_sw.js

```diff
@@ -31,6 +31,8 @@ onmessage = function (e) {
       if (err.message.includes('KeyboardInterrupt')) {
         reason = 'interrupt'
       } else {
+        err = err.toString()
+        err = err.replace(/"\<exec\>", line \d+.*(\n.*)*File "<unknown>"/, '<main.py>)')
```

Comment:
unrelated code to make the compilation error a bit shorter. (remove the bit between file <exec> and file <unknown>, which just prints the call stack for our compilation code.